plugins/nf-parquet/src/test/nextflow/parquet/DemoRecord.groovy

```diff
@@ -0,0 +1,5 @@
+package nextflow.parquet
+
+record DemoRecord(long id, String name) {
+
+}
```

Comment:
This is nice. Does groovy `record` require a specific Java version?

Reply:
it works fine with java 17 and 19 (not sure with jdk 16 but I think nobody around the world is using it)

Reply:
That's a fair assumption. Starting of Nextflow 24.11.0-edge, Java 17 is the required minimal version.